.gitignore

```diff
@@ -3,4 +3,4 @@ demo
 dist
 node_modules
 package-lock.json
-*.log
+*.log
```

Comment:
[nit] can we restore the newline at the end of this file? (ideally, it won't appear in the diff)